src/guides/json.md

```diff
@@ -0,0 +1,450 @@
+
+> The `$json` function will return an empty string if the value is a null, or the key doesn't exist, or no [`$jsonParse`](#jsonparse) function was executed, or [`$jsonClear`](#jsonclear) was executed.
+
+#### Without Arrays
+```
+$nomention
+$jsonParse[{
+    "username": "NightNutSky",
+    "tag": "6700",
+    "identity": {
+        "age": 16
+    }
+}]
+
+Username: $json[username]
+Tag: $json[tag]
+Age: $json[identity;age] years old
+```
+
+![](https://user-images.githubusercontent.com/70456337/217574893-2f962a57-f56c-49cd-8009-4dbeecbf7822.png)
+
+#### With Arrays
+```
+$nomention
+$jsonParse[{
+    "computer": [{
+        "apps": {
+            "software": ["BlueStacks", "Krita", "Visual Studio Code"\],
+            "games": ["GTA 5", "RDR 2", "CS:GO", "Cyberpunk 2077"\]
+        }
+    },{
+        "cpu": "Intel",
+        "gpu": "NVIDIA",
+        "ram": "XPG"
+    }\]
+}]
+
+Software: $json[computer;0;apps;software]
+Games: $json[computer;0;apps;games]
+Brands: CPU - $json[computer;1;cpu], GPU - $json[computer;1;gpu], RAM - $json[computer;1;ram]
+```
+
+![](https://user-images.githubusercontent.com/70456337/213921257-2f5cdcb5-8abb-45fb-8342-9512804bcb75.png)
+
+## $jsonSet
+`$jsonSet` sets or replaces the value at the specified JSON key.
+
+### Syntax
+```
+$jsonSet[Key;...;Value]
+```
+
+#### Parameters
+- `Key` `(Type: String || Flag: Required)`: The JSON key where the value should be set or replaced.
+- `Value` `(Type: Integer, Bool, Float, String || Flag: Required)`: The value which should be set or replaced.
+
+### Example
+```
+$nomention
+$jsonParse[{
+    "username": "NightNutSky",
+    "tag": "6700",
+    "identity": {
+        "age": 16
+    }
+}]
+
+Username: $json[username]
+Tag: $json[tag]
+Age: $json[identity;age] years old
+
+$jsonSet[username;Priyanuj]
+$jsonSet[tag;2626]
+$jsonSet[identity;age;19]
+
+$optOff[Username: $json[username]
+Tag: $json[tag]
+Age: $json[identity;age] years old]
+```
+
+![](https://user-images.githubusercontent.com/70456337/214134724-9ea1a4fb-177e-473f-a5ca-a326829a135d.png)
+
+## $jsonUnset
+The opposite of the [`$jsonSet`](#jsonset) function.
+
+### Syntax
+```
+$jsonUnset[Key;...]
+```
+
+#### Parameters
+- `Key` `(Type: String || Flag: Required)`: The JSON key which should be unset.
+
+### Example
+```
+$nomention
+$jsonParse[{
+    "username": "NightNutSky",
+    "tag": "6700",
+    "identity": {
+        "age": 16
+    }
+}]
+
+Username: $json[username]
+Tag: $json[tag]
+Age: $json[identity;age] years old
+
+$jsonUnset[username]
+$jsonUnset[tag]
+$jsonUnset[identity;age]
+
+$optOff[Username: $json[username]
+Tag: $json[tag]
+Age: $json[identity;age] years old]
+```
+
+![](https://user-images.githubusercontent.com/70456337/213921355-924d2647-86ff-4fc8-ac76-8edff1f82b51.png)
+
+## $jsonExists
+`$jsonExists` checks if the specified JSON key exists in the parsed JSON.
+
+> Returns an empty result if no [`$jsonParse`](#jsonparse) function was executed, or [`$jsonClear`](#jsonclear) was executed.
```

Comment:
```suggestion
> Returns an empty result if no [`$jsonParse`](#jsonparse) or [`$jsonSet`](#jsonset) function was executed, or [`$jsonClear`](#jsonclear) was executed.
```

Reply:
... ?
About it nowhere was stated.

Reply:
The description for `$jsonExists[]` states "[...] Returns an empty string if **no json function was executed** or $jsonClear was executed".
This means that if before this function no other json function that can write data was run, it will return nothing.
Mainly `$jsonParse[]` and `$jsonSet[]` are used to create a new JSON object.